echo.go

```diff
@@ -392,6 +394,20 @@ func New() (e *Echo) {
 	return
 }
 
+type TemplateRenderer struct {
+	templates *template.Template
+	renderer := &TemplateRenderer{templates: templates}
+}
+
+func (t *TemplateRenderer) Render(w io.Writer, name string, data interface{}, c echo.Context) error {
+	return t.templates.ExecuteTemplate(w, name, data)
+}
+
+func BeginTemplates(e *Echo) {
+	templates := template.Must(template.ParseGlob("templates/*.html"))
```

Comment:
* instead of `Must` (which panics) use functions that return errors and that function should return an error
* instead of hardcoded `"templates/*.html"` pass it as function parameter

Reply:
This code can be shortened to 

```go
// import 
// 	htmlTmpl "html/template"
// 	textTmpl "text/template"

type TextTemplateRenderer struct {
	Templates *textTmpl.Template
}

func (t *TextTemplateRenderer) Render(w io.Writer, name string, data interface{}, c Context) error {
	return t.Templates.ExecuteTemplate(w, name, data)
}

type HTMLTemplateRenderer struct {
	Templates *htmlTmpl.Template
}

func (t *HTMLTemplateRenderer) Render(w io.Writer, name string, data interface{}, c Context) error {
	return t.Templates.ExecuteTemplate(w, name, data)
}
```

and use as 

```go
	e.Renderer = &echo.HTMLTemplateRenderer{
		Templates: template.Must(template.ParseGlob("templates/*.html")),
	}
```

Reply:
*(I'm new in both of Go, and Echo.)*  
When I tried it as a test, I used my own repository and imported it, not compiled.  
So I don't know why it returns an error.  
This is a recommendation, this is an idea that you could implement, an idea that will make it a little bit more beginner-friendly, as the current way of middleware is not a very beginner-friendly way to make a simple app.
Thanks for reading,
@IsusRamzy

Reply:
I've allowed edits by maintainers, so you could implement it the right way.